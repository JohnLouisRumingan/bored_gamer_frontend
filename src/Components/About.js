import React from 'react'
import './css/about.css'
import { Icon, Button } from 'semantic-ui-react'

const About = () => {

    return (
        <div className='about'>
            <div className='about-content'>

                <br></br><br></br>
                <p>
                Hi, I'm John-Louis Rumingan. I made this app while discussing with my friends one day how wonderful
                it would be if we could store the games in our collection somewhere and make events with our collection.
                </p>
                <p>
                    This app allows you to create a collection, favorite games, and join game nights. 
                    It allows you to see what games are being played on game nights.
                    You can add your own games to these game nights or, if you'd like, invite others!
                </p>
                <p>
                    Created with Ruby on Rails framework for the back-end and JavaScript with React.js and Redux library
                    on the front-end. CSS used alongside with Semantic-UI-React for style.
                </p>
                <p>
                    <center><br></br>
                    Connect with me on: <br></br>
                    <Button.Group color='blue'>
                        <a href='https://www.linkedin.com/in/john-louis-rumingan/'>
                        <Button icon>
                            <Icon name="linkedin"/>
                        </Button>
                        </a>
                        <a href='https://github.com/jmr-1'>
                        <Button icon>
                            <Icon name='github'/>
                        </Button>
                        </a>
                        <a href='https://medium.com/@john.louis.rumingan'>
                        <Button icon>
                            <Icon name='medium'/>
                        </Button>
                        </a>
                    </Button.Group>
                    </center>
                </p>
            </div>
        </div>
    )
}

export default About