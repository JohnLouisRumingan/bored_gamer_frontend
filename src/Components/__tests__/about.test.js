import React from 'react'
import { shallow } from 'enzyme'
import About from '../About'

describe('About', () => {
    it('should render correctly without crashing', () => {
        const component = shallow(<About />);
        expect(component).toMatchSnapshot();
    })
})