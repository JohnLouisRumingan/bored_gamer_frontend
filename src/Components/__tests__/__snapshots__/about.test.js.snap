// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About should render correctly without crashing 1`] = `
<div
  className="about"
>
  <div
    className="about-content"
  >
    <br />
    <br />
    <p>
      Hi, I'm John-Louis Rumingan. I made this app while discussing with my friends one day how wonderful it would be if we could store the games in our collection somewhere and make events with our collection.
    </p>
    <p>
      This app allows you to create a collection, favorite games, and join game nights. It allows you to see what games are being played on game nights. You can add your own games to these game nights or, if you'd like, invite others!
    </p>
    <p>
      Created with Ruby on Rails framework for the back-end and JavaScript with React.js and Redux library on the front-end. CSS used alongside with Semantic-UI-React for style.
    </p>
    <p>
      <center>
        <br />
        Connect with me on: 
        <br />
        <ButtonGroup
          color="blue"
        >
          <a
            href="https://www.linkedin.com/in/john-louis-rumingan/"
          >
            <Button
              as="button"
              icon={true}
            >
              <Icon
                as="i"
                name="linkedin"
              />
            </Button>
          </a>
          <a
            href="https://github.com/jmr-1"
          >
            <Button
              as="button"
              icon={true}
            >
              <Icon
                as="i"
                name="github"
              />
            </Button>
          </a>
          <a
            href="https://medium.com/@john.louis.rumingan"
          >
            <Button
              as="button"
              icon={true}
            >
              <Icon
                as="i"
                name="medium"
              />
            </Button>
          </a>
        </ButtonGroup>
      </center>
    </p>
  </div>
</div>
`;
